plugins/woocommerce-blocks/assets/js/blocks/collection-filters/edit.tsx

```diff
@@ -9,55 +9,76 @@ import type { AttributeSetting } from '@woocommerce/types';
 const ATTRIBUTES = getSetting< AttributeSetting[] >( 'attributes', [] );
 
 const template = [
-	[ 'woocommerce/collection-active-filters', {} ],
 	[
-		'core/heading',
-		{
-			content: __( 'Filter by Price', 'woocommerce' ),
-			level: 3,
-		},
+		'woocommerce/collection-active-filters',
+		{},
+		[
+			[
+				'core/heading',
+				{ content: __( 'Active Filters', 'woocommerce' ), level: 3 },
+			],
+		],
```

Comment:
With the heading now becomes an inner block, I think we should move this to each filter block template instead. By doing so, when merchants insert individual block from the inserter, that filter block can also have a default title.

Reply:
Thanks for your help understanding the InnerBlocks API. I have resolved thiss in https://github.com/woocommerce/woocommerce/pull/43109/commits/c791e3a470dd3bc85037f6c5ad305b471a9c2cc6